app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -181,9 +181,25 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
         subscribeToTabCollections()
     }
 
+    private fun startOnHomeIfNeeded() {
+        val settings = context?.settings() ?: return
```

Comment:
Do we need this validation here? Maybe we could use `requireContext().settings()` instead. What do you think?

Reply:
Addressed :)